### DISCUSSION TITLE
Best use of GCODE to automate printing?

### DISCUSSION THREAD

**Discussion by MajesticLog** [blackfyre/flsun.community#55] [Q&A]
Heya,

I hope you're all well!
I'm trying to automate my FLSUN SuperRacer, and I saw that I can edit the start and end GCODE in Cura.
I found some useful commands (G4 for pausing, M26 to set the next file to read, M23 to select a file, M24 to start/resume a print, M30 to delete afile - like the one that's been printed already). I was wondering if anyone had done something similar before that I could look at?

Thank you!

**Comment by JakeOffer57** [comment]
I know this is from years ago but did you end up getting it to work? I’m trying to automate my printer using cura and I’m not sure how.
